SensorNode/sensor_nodes/remoteSensorNode.py

```diff
@@ -53,6 +53,7 @@ async def onRTPCommandResponse(self, packet: codec.binaryPacket):
                f'-f mpegts tcp://{self.data_endpoint}:{endpoint_port}')
         proc_out = asyncio.subprocess.PIPE
         proc_err = asyncio.subprocess.PIPE
+        self._log.info(f'Starting ffmpeg with command: {cmd}')
         proc = await asyncio.create_subprocess_shell(cmd,
                                                      stdout=proc_out,
                                                      stderr=proc_err)
```

Comment:
It would be good to get the `ffmpeg` error output if there is any.  See the server code.